### PR TITLE
docs: clarify alias usage for `x` parameter in vector_norm function

### LINKED ISSUE #136560: The doc of `linalg.vector_norm()` should say there is `x` or `input` parameter
### 📚 The doc issue

[The doc](https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html) of `linalg.vector_norm()` says there is `x` parameter as shown below:

> torch.linalg.vector_norm(x, ord=2, dim=None, keepdim=False, *, dtype=None, out=None) → [Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)

> x ([Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)) – tensor, flattened by default, but this behavior can be controlled using dim.

Then, `x` parameter works and `input` parameter also works as shown below:

```python
import torch
from torch import linalg

my_tensor = torch.tensor([-2., -1., 0., 1., 2., 3.])

linalg.vector_norm(x=my_tensor) # tensor(4.3589)
linalg.vector_norm(input=my_tensor) # tensor(4.3589)
```

### Suggest a potential alternative/fix

[The doc](https://pytorch.org/docs/stable/generated/torch.linalg.vector_norm.html) of `linalg.vector_norm()` should say there is `x` or `input` parameter as shown below:

> torch.linalg.vector_norm(x or input, ord=2, dim=None, keepdim=False, *, dtype=None, out=None) → [Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)

> x or input ([Tensor](https://pytorch.org/docs/stable/tensors.html#torch.Tensor)) – tensor, flattened by default, but this behavior can be controlled using dim.

### PR DESCRIPTION
**Pull request by thomasXwang**
- Added a note in the documentation specifying that the `input` parameter can be used as an alias for `x`.

Fixes #136560

### CODE REVIEW

**Review by ezyang — APPROVED** [review]

**Comment by lezcano** [comment]
nb. This is an issue with all functions in linalg. We should probably bite the bullet and just change the names of the inputs in `native_functions.yaml` for all of them (I'd hope not many people are using the keyword `input=`...).
